src/nwb_benchmarks/command_line_interface.py

```diff
@@ -36,11 +41,12 @@ def main() -> None:
         if intermediate_results_folder.exists():
             try:
                 shutil.rmtree(path=intermediate_results_folder)
-            except PermissionError:
-                raise FileExistsError(
-                    f"Unable to automatically remove {intermediate_results_folder} - please manually delete and "
-                    "try to run the benchmarks again."
+            except:
```

Comment:
I think this may have been too specific of an error redirector. Warning every time now